import io.netty.resolver.dns.DnsNameResolverException;
public class FunctionUse2980 {
public void funcUse() {
DnsNameResolverException dnsnameresolverexception = new DnsNameResolverException();
}
}